### ISSUE TITLE
A MINNT build problem with MSVCR71.dll

### ISSUE THREAD

**Issue by MasterShinker** [leecher1337/ntvdmx64#264]
Hey, I posted a comment at your YT channed that I had a problem with mktools.cmd part of building a MINNT. Actually I tried to use autobuild and it was worked fine, but I'm just interested why it doesn't works correctly if I building it manually in my VM.

Actuelly the error looks like this:
![Scrn1](https://github.com/leecher1337/ntvdmx64/assets/17530847/6201bedf-175a-4591-8a6d-93ab89e58c18)
![Scrn2](https://github.com/leecher1337/ntvdmx64/assets/17530847/cb381c50-f436-4287-9152-81f3cd0f9736)

**Comment by leecher1337** [comment]
I can only guess, maybe some issue with the path settings? Status code and error message says: `STATUS_DLL_NOT_FOUND`
Because `NTOSBE-master\tools\x86\mstools`, which also containes `cl.exe`,  should also contain mentioned `msvcr71.dll`, 
maybe it tries to use cl.exe from another path that is lacking the .dll (because maybe other path has precendence?) or it cannot find it for another reason (failed to unpack the dll to given path?).
Have you checked availability of the dll in mentioned path? Can you run cl.exe in there without issues or does it also complain when you directly run cl.exe there?

**Comment by MasterShinker** [comment]
Yeah I just got a corrupted archive of NTOSBE, not sure which site it was, now it works fine. But can I ask you, why when i'm trying to launch Doom II just two consoles appears and disappears right away? In Virtual Machine Doom II actually works, but never opens at fullscreen and no mouse capturing. And actually when I'm trying to open 16 bit windows programs like Win3.1 apps they're going to crash before they appear. Why's that?

**Comment by MasterShinker** [comment]
I'm using windows 10 x64, same as as my VM, but it works fine at VM (but without mouse capturing as I said). My friend have the same problem with NTVDMx64, he can't launch Doom too.

Btw I got WOW32 working after changing it to NTVDMx64 version, as you said. But I can open only one 16bit app. If I wanna open another one i must close Shared WOWExec that opens after I opened 16bit app or kill ntvdm.exe process in Task Manager.

**Comment by leecher1337** [comment]
Hm, tried on a freshly installed Windows 10 x64. 2 x Jewelthief and 1 x regedt16 concurrently opened without issues. You can try to use the WOW tracing capabilites described in the README, if you want to investigate further.

![image](https://github.com/user-attachments/assets/98d31d8c-2a31-4c09-9aae-762a3ee44632)
